cmd/runmqserver/process.go

```diff
@@ -44,14 +44,24 @@ func verifySingleProcess() error {
 func verifyOnlyOne(programName string) (int, error) {
 	// #nosec G104
 	out, _, _ := command.Run("ps", "-e", "--format", "cmd")
-	//if this goes wrong then assume we are the only one
-	numOfProg := strings.Count(out, programName)
+	lines := strings.Split(out, "\n")
```

Comment:
I think it would be easier to change `cmd` to `comm` instead.

Reply:
I'm not familiar with `comm`.

Reply:
Can confirm that simply switching to comm format should give the desired result:
```
sh-4.4$ runmqdevserver > /dev/null 2>&1 &
[1] 58
sh-4.4$ ps -e --format comm
COMMAND
sh
runmqdevserver
ps
sh-4.4$ ps -e --format cmd
CMD
/usr/bin/qemu-x86_64 /bin/sh /bin/sh
/usr/bin/qemu-x86_64 /usr/local/bin/runmqdevserver runmqdevserver
ps -e --format cmd
```

Reply:
Thanks. I've opened a new PR with the simpler fix https://github.com/ibm-messaging/mq-container/pull/527